### PR TITLE
add com.adilhanney.timing

### PR DESCRIPTION
**Pull request by adil192**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  <s>If not, the unredistributable parts are using an extra-data source type.</s>
- [x] I am an upstream contributor to the project. <s>If not, I contacted upstream developers about submitting their software to Flathub. **Link:**</s>
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.adilhanney.timing.timing_flutter.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        "mkdir -p TimingFlutter",
+        "tar -xf TimingFlutter-Linux-Portable.tar.gz -C TimingFlutter",
+
+        "cp -r TimingFlutter /app/",
+        "chmod +x /app/TimingFlutter/timing_flutter",
+        "mkdir -p /app/bin",
+        "ln -s /app/TimingFlutter/timing_flutter /app/bin/timing_flutter",
+
+        "mkdir -p /app/share/metainfo",
+        "cp com.adilhanney.timing.timing_flutter.metainfo.xml /app/share/metainfo/",
```
Comment (hfiguiere) [inline com.adilhanney.timing.timing_flutter.json:32]:
```suggestion
        "install -Dm644 com.adilhanney.timing.timing_flutter.metainfo.xml -t /app/share/metainfo/",
```

File: com.adilhanney.timing.timing_flutter.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+        "mkdir -p /app/share/icons/hicolor/scalable/apps",
+        "cp icon.svg /app/share/icons/hicolor/scalable/apps/com.adilhanney.timing.timing_flutter.svg",
```
Comment (hfiguiere) [inline com.adilhanney.timing.timing_flutter.json:35]:
```suggestion
        "install -Dm644 icon.svg -t /app/share/icons/hicolor/scalable/apps/com.adilhanney.timing.timing_flutter.svg",
```

File: com.adilhanney.timing.timing_flutter.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+        "mkdir -p /app/share/applications",
+        "cp com.adilhanney.timing.timing_flutter.desktop /app/share/applications/",
+
+        "mkdir -p /app/share/appdata",
+        "cp com.adilhanney.timing.timing_flutter.metainfo.xml /app/share/appdata/"
```
Comment (hfiguiere) [inline com.adilhanney.timing.timing_flutter.json:41]:
```suggestion
        "install -Dm644 com.adilhanney.timing.timing_flutter.desktop -t /app/share/applications/"
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build com.adilhanney.timing.timing_flutter

File: com.adilhanney.timing.timing_flutter.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
+{
+  "app-id": "com.adilhanney.timing.timing_flutter",
```
Comment (hfiguiere) [inline com.adilhanney.timing.timing_flutter.json:2]: I would just use `com.adilhanney.timing` (granted that the domain is actually yours). But the website at the github repo point to https://adil.hanney.org/timing_flutter/

**Comment by nedrichards** [comment]
bot, build com.adilhanney.timing

**Comment by adil192** [comment]
bot, build com.adilhanney.timing

**Pull request closed without merging**